resources/checkDefinitions.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs').promises;
+const { exec } = require('child_process');
+(async () => {
+  const dir = await fs.readdir('./');
+  console.log(dir);
+  const [currentDefinitions, currentDocs] = await Promise.all([
+    fs.readFile('./src/options/Definitions.js', 'utf8'),
+    fs.readFile('./src/options/Docs.js', 'utf8'),
+  ]);
+  exec('npm run definitions');
+  await new Promise(resolve => setTimeout(resolve, 2000));
+  const [nowDefinitions, nowDocs] = await Promise.all([
```

Comment:
Could you rename this to `new...` instead of `now...`? I think it sounds slightly more descriptive than a time reference.